rust/tests/namespace.ts

```diff
@@ -6,12 +6,14 @@ import {
   IDL as NamespaceProgramIDL,
 } from "../target/types/namespace";
 import { Item, IDL as ItemProgramIDL } from "../target/types/item";
+import { Matches, IDL as MatchesProgramIDL } from "../target/types/matches";
 import { NamespaceProgram } from "../../js/lib/src/contract/namespace";
 import * as nsIx from "../../js/lib/src/instructions/namespace";
 import * as nsState from "../../js/lib/src/state/namespace";
 import * as pids from "../../js/lib/src/constants/programIds";
 import * as pdas from "../../js/lib/src/utils/pda";
```

Comment:
Update all these imports to use the package, not a direct file import.

Reply:
I'm not sure I can change this until this is merged otherwise the tests will break?

Reply:
The turborepo changes that were added to this repo should no longer cause that to be breaking to use the package names.

https://github.com/raindrops-protocol/raindrops/pull/29

Reply:
I had to change the Match IDL for my integration, so I think the turbo repo one would be out of date for this PR